README.md

```diff
@@ -29,7 +29,7 @@ truffle migrate --network sapphire-mainnet
 ```
 =====================
 
-   Deploying 'WROSE9'
+   Deploying 'WROSE'
```

Comment:
The contract hasn't been re-deployed though...

Reply:
This is just an example